pkgs/tools/cd-dvd/ventoy-bin/default.nix

```diff
@@ -79,13 +79,13 @@ in stdenv.mkDerivation rec {
              ExtendPersistentImg.sh_ventoy-extend-persistent; do
         makeWrapper "$VENTOY_PATH/''${f%_*}" "$out/bin/''${f#*_}" \
                     --prefix PATH : "${lib.makeBinPath buildInputs}" \
-                    --run "cd '$VENTOY_PATH' || exit 1"
+                    --chdir "$VENTOY_PATH"
```

Comment:
This previously relied upon runtime environment variable expansion of `$VENTOY_PATH`. This will now instead expand at build time.

Reply:
Nope. The outer quotes are double so variables are expanded.

```
~ $ foo=bar; echo "cd '$foo'"
cd 'bar'
```

Reply:
You are right